src/lang/ts/intermock.ts

```diff
@@ -40,16 +40,14 @@ export interface Options {
 
   // One of object|json|string. Strings have their object's functions
   // stringified.
-  output?: OutputType;
+  output?: string;
```

Comment:
could also export OutputType to use in cli/index.js

Reply:
Output type is still useful because Intermock can generate more than just stringified data. With an `object` type, it can actually generate mock functions via the `new Function` constructor.

Reply:
Added it back in, and used it to type output in the cli